app/routes/docs.circulars.corrections/route.mdx

```diff
@@ -0,0 +1,54 @@
+---
+handle:
+  breadcrumb: Corrections
+---
+
+import requestbuttonscreenshot from './request_correction_button.png'
+import versionbuttonscreenshot from './version_button.png'
+
+# Corrections
+
+Circulars authors can request edits to their own archived Circulars to correct inaccurate information, especially when the correction is important for the follow-up community's activities. These may include author lists, subject corrections, typos, and incorrect citations. This capability should not be used to add details of new observations or analyses — instead users should submit new Circulars with appropriate subjects (e.g. "refined analysis" or "additional observations").
+
+## Who can request corrections?
+
+Anyone with a [peer endorsement](/docs/circulars/submitting#to-request-a-peer-endorsement) can submit requests for corrections. If the requester is the submitting author, the moderator will likely approve the request immediately. If the requester is a co-author or community member, the moderator will get concurrence from the submitting author via email.
+
+## How to request corrections?
+
+- [Sign in to your GCN account](/login)
+- Navigate to the [archive page](/circulars) for that particular circular.
+- Select "Request Correction"
+- Edit the Circular as needed, and "Submit".
+- The GCN team moderators will review the requested correction, and may contact the submitter for their concurrence.
+
+<img
+  src={requestbuttonscreenshot}
+  alt="Screen shot of Circulars archive menu showing how to request corrections."
+  width="530"
+  height="49"
+  className="height-auto"
+/>
+
+## Request approval workflow
+
+- After a requester submits a corrections request, the GCN team moderators are informed via the GCN helpdesk ticketing system.
+- The requester will receive an email from support@gcn.nasa.gov acknowledging that the ticket was created.
+- The moderator will either immediately approve or reject the request, or contact the original submitter (if different from the request submitter) via email for clarification or concurrence before accepting or rejecting the request.
+- The requester will receive an email when the ticket is closed.
+
+## View the revision history
+
+A version history for revised Circulars is publicly viewable in the archive with the "Versions" dropdown. Information about the revision are added to the Circular header including when it was edited, who requested the edit, and which GCN moderator implemented the edits.
+
```

Comment:
Please also include a comment in here about the url of the latest version. The path https://gcn.nasa.gov/circulars/{circularID} will always point to the latest version of the circular